portfolio/src/main/webapp/style.css

```diff
@@ -16,10 +16,11 @@ body {
   font-family: Poppins;
   font-weight: 400;
   color: white;
+  background: black;
 }
 
 html { 
-  background: url("/images/portfolioMainPageGIF.gif") no-repeat center center fixed; 
+  /*background: url("/images/portfolioMainPageGIF.gif") no-repeat center center fixed; */
```

Comment:
Why is this commented out? It's probably best to remove if it isn't being used.

Reply:
makes sense, fixed